web/pages/article/article.module.css

```
@@ -1,20 +1,112 @@
+.extraHeroContent {
+  margin-top: 48px;
+}
+
+.authors {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.author {
+  display: inline-block;
+}
+
+.timestamps {
+  margin-top: 24px;
+  font: var(--font-ui-xs);
+  letter-spacing: var(--letter-spacing-ui-xs);
+}
+
+.publishedAt,
+.updatedAt {
+  margin: 0;
+}
+
 .container {
   margin: 80px 0;
 }
 
+.asides {
+  margin-top: 80px;
+}
+
+.relatedItems:nth-child(n + 2) {
+  margin-top: 32px;
+}
+
+.relatedItemsHeading {
+  font: var(--font-ui-xs-medium);
+  letter-spacing: var(--letter-spacing-ui-xs);
+  margin: 0 0 10px;
+}
+
+.relatedItemsList {
+  padding: 0;
+  margin: 0;
+}
+
+.relatedItem {
+  display: inline-block;
+}
+
 @media (--medium-up) {
+  .extraHeroContent {
+    margin-top: 64px;
+  }
+
+  .publishedAt,
+  .updatedAt {
+    display: inline;
+  }
+
+  .updatedAt::before {
```

After commit 3057fc620877bbb43aa7a2492780f3f865237fb0, it is no longer guaranteed that there is a publishedAt.

Suggestion (not tested):

```suggestion
  .publishedAt + .updatedAt::before {
```

---

Tested all permutations locally, it worked well. Thank you!